contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -139,8 +143,10 @@ impl Contract {
                 Some(amount) => amount,
```

Comment:
isn't `PLAYERS_NUM` always 2? Maybe we can remove that const, and just use `2`, and the Rust will figure out the typee.